.github/workflows/docs-build.yml

```
@@ -49,6 +49,9 @@ jobs:
 
       - name: Build and test Sphinx docs
         run: |
+          sudo add-apt-repository ppa:wrathfulspatula/vm6502q
+          sudo apt update
+          sudo apt install libqrack-dev
```

```suggestion
          sudo apt update
          sudo install ocl-icd-opencl-dev
```

Once you have `pyqrack` in the Python requirements, which you do, it's actually enough to install libOpenCL, you don't need libqrack from PPA. I figured this out by
1. running the workflow locally with the `act` tool (`act push --container-architecture linux/amd64 -j docs`)
2. entering the docker container spawned by 1. (`docker exec -it <container_id> bash`)
3. open a python console
4. run `from ctypes import *; CDLL("/usr/local/lib/python3.10/dist-packages/pyqrack/qrack_system/qrack_lib/libqrack_pinvoke.so")`, and observe the error
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.10/ctypes/__init__.py", line 374, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: libOpenCL.so.1: cannot open shared object file: No such file or directory
```
due to a missing libOpenCL library in the container. (cc @natestemen)

---

Was able to use `pyqrack` inside the docs build by doing the following:

1. mitiq/requirements/requirements-dev.txt include pyqrack to the dependencies. (pyqrack==1.30.0)
2. mitiq/.github/workflows/docs-build.yml use apt to install opencl.
```
sudo apt update
sudo apt install ocl-icd-opencl-dev
```
3. mitiq/.readthedocs.yml add the apt_package to the build.
```
apt_packages:
  - ocl-icd-opencl-dev
  ```

---

@bdg221 Nice! 🎉 
Special shoutout to @natestemen for bringing the topic to the Mitiq coding session.